aiohue/v1/__init__.py

```diff
@@ -16,7 +16,9 @@
 from .scenes import Scenes
 from .sensors import Sensors
 
-MAX_RETRIES = 25  # how many times do we retry on a 503 (bridge overload/rate limit)
+MAX_RETRIES = (
+    25  # how many times do we retry on a 503 or 429 (bridge overload/rate limit)
```

Comment:
can you put the comment above the constant definition please ? that looks more clear.